src/test/regress/expected/multi_router_planner_fast_path.out

```diff
@@ -2111,6 +2111,26 @@ DEBUG:  query has a single distribution column value: 4
      0
 (1 row)
 
+-- test INSERT using values from generate_series() and repeat() functions
+INSERT INTO authors_reference (id, name) VALUES (generate_series(1, 10), repeat('Migjeni', 3));
+DEBUG:  Creating router plan
+SELECT * FROM authors_reference;
```

Comment:
We should add `ORDER BY` to make sure the output doesn't change on different runs of the test

Reply:
will do :+1: